ci/gpu/build.sh

```diff
@@ -54,14 +54,21 @@ conda install -c conda-forge -c rapidsai -c rapidsai-nightly -c rapidsai/label/x
       "umap-learn" \
       "protobuf >=3.4.1,<4.0.0" \
       "nccl>=2.4" \
-      "dask=2.8*" \
-      "distributed=2.8*" \
+      "dask>=2.8.0" \
```

Comment:
If I understand this flow correctly, and from looking at the CI logs, this gets installed and then immediately overridden by the source build from master. Do we still need to explicitly install it here?

Reply:
This forces all of the dask dependencies to get conda installed though

Reply:
Yes, it was the comment by @kkraus14 above, so that the dependencies are fetched

Reply:
I notice it was removed from the conda dev env yaml files altogether. If we are able to remove them from this script as well, it would be one less thing to have to change whenever we do bump versions.

Reply:
I figured it might have been the case, but wasn't sure if the pip install from the source would have done this as well. Could this still fail if a dependency changes in master?

Reply:
yes while there is a lag between master having a dependency change and the dask conda package having the new requirement, so we would need to add the dependency manually. That said, cuDF has it pinned to 2.1, so it seems like it hasn't changed requirements much

Reply:
yes while there is a lag between master having a dependency change and the dask conda package having the new requirement, so we would need to add the dependency manually. That said, cuDF has it pinned to 2.1, so it seems like it hasn't changed requirements much

Reply:
Sounds good.

Reply:
yes while there is a lag between master having a dependency change and the dask conda package having the new requirement, so we would need to add the dependency manually. That said, cuDF has it pinned to 2.1, so it seems like it hasn't changed requirements much